src/providers/fail_over.c

```diff
@@ -1188,9 +1189,18 @@ fo_resolve_service_done(struct tevent_req *subreq)
                                     &common->rhostent);
     talloc_zfree(subreq);
     if (ret != EOK) {
-        DEBUG(SSSDBG_CRIT_FAILURE, "Failed to resolve server '%s': %s\n",
+        if (resolv_status == ARES_EFILE)
+        {
```

Comment:
Style - `{` should be in the same line as `if` [1]

[1] https://sssd.io/contrib/coding-style.html